checks/remotesettings/total_approvals.py

```
@@ -0,0 +1,87 @@
+"""
+Dummy-check to obtain information about the total number of approvals in the last days.
+
+For each day, the total number of approvals is returned, with details per collection.
+"""
+from collections import Counter
+from datetime import datetime, timezone
+
+from telescope.typings import CheckResult
+from telescope.utils import run_parallel, utcfromtimestamp, utcnow
+
+from .utils import KintoClient, fetch_signed_resources
+
+
+ONE_DAY_MSEC = 24 * 3600 * 1000
+
+DEFAULT_PLOT = "0.totals"
+
+
+async def get_approvals(client, bucket, min_timestamp, max_timestamp):
```

Should we throw an error if `min_timestamp > max_timestamp`?